### ISSUE TITLE
Questions about the inference code （need  model.eval()）

### ISSUE THREAD

**Issue by BaofengZan** [sair-lab/AirLine#8]
I use the ‘Airline_demo.py’ to test the images and the original file can get normal results.  But when I add Premodel.eval(), the result is 0（other parameters remain the same）.   

without model.eval()
![image](https://github.com/sair-lab/AirLine/assets/20653176/36c36ee9-c23e-412e-8ee5-93453dc630db)

with model.eval()
![image](https://github.com/sair-lab/AirLine/assets/20653176/58553f6d-a949-41a0-a23a-d4b92a44e479)

**Comment by Lx017** [comment]
hmmmm, I had not tested this though, this may affect some normalization but I am not sure. if no .eval() works then I recommend to just leave it so?

**Comment by BaofengZan** [comment]
When I export onnx using torch.onnx.export,  the bn layer exports in eval mode by default，  If the onnxruntime inference is used, the result is incorrect.       But when I export the bn layer in TRAINING mode， when converting to tensorrt, it will prompts that the BN layer is in TRAINING mode, which makes it impossible to convert. So I think I still need to check this issue.

Error when converting tensorrt。Official Answers：  https://github.com/NVIDIA/TensorRT/issues/3457#issuecomment-1817477998
![image](https://github.com/sair-lab/AirLine/assets/20653176/3bbc4157-6c15-4aef-be4a-727dbe2a045c)

**Comment by Lx017** [comment]
yeah sorry, that is indeed something we missed...

**Comment by BaofengZan** [comment]
Thank you, I look forward to you being able to solve this problem.
